tensorflow_lite_support/ios/TensorFlowLiteTaskVisionObjCPP.podspec

```diff
@@ -0,0 +1,20 @@
+Pod::Spec.new do |s|
+  s.name             = 'TensorFlowLiteTaskVisionObjCPP'
```

Comment:
Instead of having a dedicated podspec for the ObjC++ implementation, can you add the `TensorFlowLiteTaskVisionObjCPP.framework` as a vendored framework for the current TensorFlowLiteTaskVision pod? We don't want to expose the internal structure of the framework to the end-users if possible.

Reply:
Thanks Prianka! LGTM.

@schmidt-sebastian @lu-wang-g Do you have any feedback?

Reply:
@priankakariatyml Can you check if the new setup can make the Vision pod and Text pod co-exist in the same project? I guess that you may tweak the code to hide the duplicating symbols.